static/login.css

```
@@ -0,0 +1,12 @@
+/*Body styling*/
+body {
+  background-image: url("../static/img1.jpg");
+  background-color: rgb(48, 44, 44);
+  height: 1000px;
```

This should probably be changed to be relative to whatever device the site is being viewed on. If this is supposed to cover the entire screen, I'd recommend changing it to this:
```suggestion
  height: 100vh;
```